### ISSUE TITLE
heading-capitalization (MD063) - false positive when heading contains a pronoun I

### ISSUE THREAD

**Issue by adam-moss** [rvben/rumdl#845]
### rumdl version

0.2.63

### Installation method

mise

### Bug description

When a heading contains a pronoun I and `style` is set to `sentence-case`, it incorrectly suggests lowercasing the I

### Steps to reproduce

```markdown
# How do I debug playbooks?
```

### Expected behavior

No error reported.

### Actual behavior

```text
[MD063] Heading should use sentence case: 'How do I debug playbooks?' -> 'How do i debug playbooks?'
```

### Configuration

```toml
[MD063]
style = "sentence-case"
preserve-cased-words = true
```

### Operating system

macOS

### Additional context

Adding 

`ignore-words = ["I", "I'd", "I'll", "I'm", "I've"]` to the configuration fixes it, but this should likely be a default

**Comment by rvben** [comment]
Thanks @adam-moss, and your suggestion is what shipped: this is the default now rather than something you configure around.

Sentence case preserves the first-person `I`, including `I'd`/`I'll`/`I'm`/`I've`, recognized only at a real word boundary so `# Iterating on ideas` is untouched, and an explicit `ignore-words` still wins. You can drop the workaround entry. Fixed in v0.2.65.
